server/src/modules/mentorship-response/mentorship-response.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { MentorshipResponseDTOGet } from './mentorship-response';
+import {
+  MentorshipRequest,
+  RequestStatus,
+} from '../mentorship-request/mentorship-request.model';
+import {
+  MentorshipResponse,
+  MentorshipResponseDecision,
+} from './mentorship-response.model';
+
+@Injectable()
+export class MentorshipResponseService {
+  async getResponsesForMentee(
+    menteeId: number,
+  ): Promise<MentorshipResponseDTOGet[] | null> {
```

Comment:
nit: can this just be `Promise<MentorshipResponseDTOGet[]>`? I don't think you ever return `null`